modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -489,6 +491,12 @@ rtfctl get secure-properties
 rtfctl get mule-license
 ----
 
+* Displays registry credentials
```

Comment:
@IsaacEldridge can you please add **rtf** registry credentials. So that we specifically convey that they are for rtf registry.